rero/rero-ils
# tests: fix travis trouble with pytest-invenio

Last days a [new pytest-invenio 1.2.2 version](https://github.com/inveniosoftware/pytest-invenio/commit/9a5d15bbfeda699bcc67805519861409c567c6f0) was released.
It downgrades our pytest-flask from 1.0.0 to 0.15.1 and our Flask from
1.1.2 to 1.04.
These changes break our tests. Mainly with Flask and werkzeug problems.
Cf. https://travis-ci.org/github/blankoworld/rero-ils/jobs/684571964#L1905

For now, this commit limits tests problem by keeping our pytest-invenio
version, pytest-flask and Flask.

* Fix pytest-invenio to 1.2.1 to avoid problems in our tests

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

This should fix the issue about tests like here: https://travis-ci.org/github/rero/rero-ils/jobs/684554447

## How to test?

* Check Travis tests
* Check changes in Pipefile and Pipfile.lock

## Code review check list
